src/details/ArborX_DetailsDistributedTreeImpl.hpp

```diff
@@ -298,8 +298,9 @@ DistributedTreeImpl<DeviceType>::sendAcrossNetwork(
   {
     // For multi-dimensional views, we need to first copy into a separate
     // storage because of a different layout
-    auto tmp_view =
-        Kokkos::create_mirror_view_and_copy(space, imports_layout_right);
+    auto tmp_view = Kokkos::create_mirror_view_and_copy(
+        Kokkos::view_alloc(space, typename ExecutionSpace::memory_space{}),
```

Comment:
Does it need `WithoutInitializing`, or is that done by default?

Reply:
That's done by default.